README.rst

```diff
@@ -21,6 +21,7 @@ If you want the output stored to a file you can specify a path for this.
     - uses: twisted/python-info-action@v1.0.1
       with:
         output-path: environment.log
+        python-path: env/*/python
```

Comment:
Maybe is best to have a separate example

```
You can call the action using a specific Python version, by specifying the full path to the Python executable.
You can also pass a glob expression::
```

And then the example.

Reply:
Agreed that it should be separate.  Will correct.